src/dmd/statementsem.d

```diff
@@ -2444,13 +2444,16 @@ else
                 else if (e.isBool(false))
                     inlining = PINLINE.never;
 
-                    FuncDeclaration fd = sc.func;
+                FuncDeclaration fd = sc.func;
                 if (!fd)
                 {
                     ps.error("`pragma(inline)` is not inside a function");
                     return setError();
                 }
-                fd.inlining = inlining;
+                if (!global.params.cov) // inline except when doing coverage
```

Comment:
it would look better if you checked for cov being true,
an in that case set inline to NOinline (or whatever it is.).

Reply:
Done.

Reply:
> it would look better if you checked for cov being true,
> an in that case set inline to PINLINE.never;

This will result in `pragma(line, ...)`-tagged functions having `PINLINE.never` and other functions have `PINLINE.default`. Is this really what we want?

Reply:
I don't think it matters.
This is a hack in any case.